src/main/java/frc/team166/robot/Robot.java

```diff
@@ -132,8 +136,90 @@ public void testPeriodic() {
     }
 
     public Command CrossLineAndDropCube() {
-        return new CommandChain("Cross Line And Drop Cube").then(drive.DriveTime(3, 0.6), lift.RaiseLiftALittle())
-                .then(manipulator.CubeEject());
+        // return new CommandChain("Cross Line And Drop Cube").then(drive.DriveTime(1.8, 0.6), lift.LowerLiftForTime())
+        //       .then(lift.RaiseLiftALittle()).then(manipulator.CubeEject());
+        return new CommandChain("Cross Line And Drop Cube").then(lift.LowerLiftToLimitSwitch());
+    }
+
+    public Command MidAuto() {
+        String gameData;
+
+        gameData = DriverStation.getInstance().getGameSpecificMessage();
+        double degrees = 0.0;
+        if (gameData.length() > 0) {
```

Comment:
I just realized that gamedata will likely be empty when this function is called (At robot start time) As such the commands returned by this and the two scale functions will not be what is expected. The game data needs to be looked at during the execution of a command to ensure that the match has started and game data is available.